pyproject.toml

```
@@ -66,6 +66,9 @@ dev-dependencies = [
     "pytest-asyncio>=1.0.0",
     "pytest-mock>=3.14.1",
     "ruff>=0.11.12",
+    "langchain-core>=0.1.0",
+    "langchain-experimental>=0.0.50",
+    "langchain-community>=0.0.20",
 ]
```

```suggestion
    "fastapi>=0.115.0",
    "uvicorn[standard]>=0.34.0",
    "ag-ui-protocol>=0.1.4",
    "aiofiles>=23.2.1",
    "filetype>=1.2.0",
    "httpx>=0.28.0",
]
```

And in the Makefile, please add this:

```
@@ -1,28 +1,28 @@
 .PHONY: sync
 sync:
-	uv sync --all-extras
+	uv sync --all-extras --dev
```

Then run `make sync` locally to regenerate uv.lock.